chart/values.yaml

```diff
@@ -1695,7 +1695,7 @@ config:
     # For Airflow 1.10
     rbac: 'True'
   celery:
-    flower_url_prefix: '{{ .Values.ingress.flower.path }}'
+    flower_url_prefix: ''
```

Comment:
Isn't something like `{{ .Values.ingress.flower.path | default "" }` better here?

Reply:
But it will always select .Values.ingress.flower.path if it is not empty. That means, it will select "/" always which doesn't work. url_prefix should be "" instead. Is that correct?

Reply:
> But it will always select .Values.ingress.flower.path if it is not empty. That means, it will select "/" always which doesn't work. url_prefix should be "" instead. Is that correct?

Agree it. `{{ .Values.ingress.flower.path | default "" }` may not work. Because `.Values.ingress.flower.path` default value must be `/`, can not use empty string.
ref: https://github.com/apache/airflow/pull/27472#pullrequestreview-1166214946
@XD-DENG

Reply:
I believe I must have missed something, but may you help me understand why the change here is to propose defaults to an empty string, while you think `{{ .Values.ingress.flower.path | default "" }` may not work?

Reply:
We're making url_prefix default to "" because if we take `{{ .Values.ingress.flower.path }` it adds a '/' in front of the script paths to load as mentioned in the issue, which comes from the value of `{{ .Values.ingress.flower.path }`. which in turn doesn't add localhost to requests in browser and try to make direct requests to static assets. so it gives 404.
Making it "" removes that extra '/' and the path works correct.

Reply:
I believe the current state of PR will work fine for us.